shared/src/packages/orders/libs/types/order-find-by-id-response-dto.type.ts

```diff
@@ -0,0 +1,15 @@
+import { type DriverDto } from '~/packages/drivers/drivers.js';
+import { type UserEntityT } from '~/packages/users/users.js';
+
+import { type OrderEntity } from './order-entity.type.js';
+
+type OrderFindByIdResponseDto = OrderEntity & {
+  driver: Pick<DriverDto, 'driverLicenseNumber'> & {
+    user: {
+      firstName: UserEntityT['firstName'];
+      lastName: UserEntityT['lastName'];
+    };
+  };
```

Comment:
I meant to do like this, to avoid such structure driver: { user: fields } }
```suggestion
  driver: DriverDto;
  };
```